Makefile

```diff
@@ -21,8 +21,11 @@ ENVTEST_K8S_VERSION = 1.26
 # - use environment variables to overwrite this value (e.g export VERSION=0.0.2)
 DEFAULT_VERSION := 0.0.1
 # Let CI set VERSION based on git tags. But heads up, VERSION should not have the 'v' prefix!
-VERSION ?= $(DEFAULT_VERSION)
-export VERSION
+export VERSION ?= $(DEFAULT_VERSION)
+# For the replaces field in the CSV, mandatory to be set for versioned builds! Should also not have the 'v' prefix.
+export PREVIOUS_VERSION ?= $(DEFAULT_VERSION)
+# Lower bound for the skipRange field in the CSV, should be set to the oldest supported version
+export SKIP_RANGE_LOWER ?= "0.1.0"
```

Comment:
Any reason those 3 fields need to be exported ?

Reply:
I made bad experience with vars not being set correctly when used in Makefiles / targets / scripts called by targets, so it's just "better safe than sorry" 🙂 Might that it's not needed.